HelpSource/Guides/UsingQuarks.schelp

```diff
@@ -179,7 +179,9 @@ section:: Publishing a Quark
 
 Make a git repository and push it to github, gitlab, bitbucket or any publicly accessible git host.
 
-Be sure to strong::tag your releases::. The name of a tag should be the release version to which it corresponds. Using semantic versioning (eg "v1.0.1") is strongly recommended. This provides the Quarks system with the ability to list available versions and allow the user to choose which version to checkout. Version tags also allow other quark developpers to pin their dependencies to specific release versions.
+Be sure to strong::tag your releases::. The name of a tag should be the release version to which it corresponds.
+Using Semantic Versioning footnote::For information on Semantic Versioning see https://semver.org:: (eg "v1.0.1") is strongly recommended.
```

Comment:
why a footnote rather than inline link?

Reply:
because of the parenthesis that's there `(v1.0.1)`. would you prefer I make the words `Semantic Versioning` into a link to semver.org? I could do that instead.

Reply:
yeah! that would be more straightforward i think, that's what we usually do in other files :)

Reply:
done!